CONVENTIONS.md

```diff
@@ -0,0 +1,45 @@
+# Convetions
+
+This document gives coding conventions for the Superface capabilities created in [Station repository](https://github.com/superfaceai/station).
+
+## Profile
+
+### Structure
+
+- Use 2 spaces per indentation level.
+- Profile should contain one usecase.
```

Comment:
Maybe let's follow https://datatracker.ietf.org/doc/html/rfc2119 and use all capitals: 

> - Profile SHOULD contain one usecase.
> - Usecase MUST define safety.